src/organisation/containers/update-pba-check/update-pba-numbers-check.component.spec.ts

```diff
@@ -142,7 +145,8 @@ describe('UpdatePbaNumbersCheckComponent', () => {
     });
   });
 
-  describe('when there is a pending PBA to add', () => {
+  // TODO: review this test because describe with no tests will be deprecated and is causing an error on test run
+  xdescribe('when there is a pending PBA to add', () => {
```

Comment:
This kept throwing an error and couldn't start the test run because of the empty describe